### DISCUSSION TITLE
关于SEVI大作业中“间隙距离”

### DISCUSSION THREAD

**Discussion by jiqingsong** [physics-data/faq#93] [Q&A] [answered]
这个问题来源于level3的题目叙述
![image](https://user-images.githubusercontent.com/37206109/131490318-9210920e-c5bc-4a48-86e5-be743c42bdc1.png)
这里究竟是距离还是时间？
若是距离，则电子的整体飘逸速度是多少？
若没有整体飘逸速度，则是否意味着一部分电子将向-y方向飞出，而无法进入下一个MCP或屏上？

**Comment by jiqingsong** [comment]
另外，关于要求的“完备描述”，是否可以理解为，有了这些信息，可以通过计算完整得到所有需要的结果

**Reply by littlewu2508** [reply]
> 另外，关于要求的“完备描述”，是否可以理解为，有了这些信息，可以通过计算完整得到所有需要的结果

正是如此

**Comment by littlewu2508** [comment, marked as answer]
> 这个问题来源于level3的题目叙述
> ![image](https://user-images.githubusercontent.com/37206109/131490318-9210920e-c5bc-4a48-86e5-be743c42bdc1.png)
> 这里究竟是距离还是时间？
> 若是距离，则电子的整体飘逸速度是多少？
> 若没有整体飘逸速度，则是否意味着一部分电子将向-y方向飞出，而无法进入下一个MCP或屏上？

是距离。电子的速度不重要，你可以认为电子是直线运动，速度无限快。

确实有电子可能会沿着 -y 飞出。不过概率很小了；MCP 的尺寸也与荧光屏一样

**Reply by jiqingsong** [reply]
概率为什么会很小呢，MCP生成电子的分布在phi方向上均匀，即使不考虑theta，也会有一半的电子无法到达下一个屏幕？

**Reply by littlewu2508** [reply]
这个角分布的球坐标系极轴是 MCP 平面的法线（参见 https://github.com/physics-data/faq/discussions/85 ）

**Reply by jiqingsong** [reply]
> 这个角分布的球坐标系极轴是 MCP 平面的法线（参见 #85 ）

其他的角分布的极轴是z轴，level3MCP生成的角分布的极轴是MCP法线方向，是这样理解吗？

**Reply by littlewu2508** [reply]
> level3MCP生成的角分布的极轴是MCP法线方向

是的

>其他的角分布的极轴是z轴

如果说 “其它” 指的是初始光电子的动量角分布，那的确
